src/lib.rs

```diff
@@ -1,2 +1,396 @@
 extern crate libc;
-include!(concat!(env!("CARGO_MANIFEST_DIR"), "/src/arrayfire.rs"));
\ No newline at end of file
+
+use std::fmt;
+use libc::c_void;
+use libc::c_int;
+use libc::c_uint;
+use libc::c_double;
+use libc::c_longlong;
+
+use std::ops::Index;
+use std::ops::Add;
+
+#[link(name="afcpu")]
```

Comment:
I removed link attribute on my machine and it worked fine. Internet says gcc does this magic, picking symbols automatically. So i think we can avoid this. Not sure how this will work on windows though.